libafl/src/stages/sync.rs

```diff
@@ -105,6 +106,19 @@ where
                     .unwrap()
                     .last_time = max_time;
             }
+            // Iterate over the paths of files left to sync.
+            // By keeping track of these files, we ensure that no file is missed during synchronization,
+            // even in the event of a target restart.
+            let to_sync = self.to_sync.clone();
```

Comment:
Shouldn't these be stored in the metadata? If the input crashes, the set will be empty on reload, right?